darts/models/forecasting/regression_ensemble_model.py

```diff
@@ -22,7 +19,7 @@
 class RegressionEnsembleModel(EnsembleModel):
     def __init__(
         self,
-        forecasting_models: Union[List[ForecastingModel], List[GlobalForecastingModel]],
+        forecasting_models: List[ForecastingModel],
```

Comment:
Note: that's not exactly the same thing as `Union[List[LocalForecastingModel], List[GlobalForecastingModel]]`

Reply:
Oh yeah, good catch!